replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/mcard/ReplicationHistoryAttributes.java

```diff
@@ -123,7 +122,13 @@
             false /* multivalued */,
             BasicTypes.STRING_TYPE));
     descriptors.add(
-        new ReplicationConfigAttributes().getAttributeDescriptor(ReplicationConfig.NAME));
+        new AttributeDescriptorImpl(
```

Comment:
We shouldn't need this file at all anymore since we are not storing anything as metacards right?